client/src/components/EventCard.vue

```diff
@@ -2,13 +2,15 @@
   <v-card class="mx-3">
     <v-card-title>
       <span class="font-weight-bold">{{ event.name }}</span>
-      <v-icon v-if="isTeamAdmin && event.isPrivate" icon="mdi-pencil" @click="openModal"></v-icon>
+      <v-icon v-if="isTeamAdmin && !event.isPublic" icon="mdi-pencil" @click="openModal"></v-icon>
       <v-spacer></v-spacer>
       <v-chip
         variant="outlined"
-        :class="{ 'text-secondaryGreen': event.isPrivate, 'text-secondaryBlue': !event.isPrivate }"
-        >{{ event.isPrivate ? 'Private' : 'Official' }}</v-chip
+        :class="{ 'text-secondaryGreen': !event.isPublic, 'text-secondaryBlue': event.isPublic }"
       >
+        {{ !event.isPublic ? 'Private' : 'Official' }}
```

Comment:
Change to this so its easier to read

{{ event.isPublic ? 'Official' : 'Private' }}